README.md

```diff
@@ -2,4 +2,25 @@
 
 This project is to demonstrate how [Airwallex platform onboarding SDK](https://www.npmjs.com/package/@airwallex/platform-onboarding-sdk?activeTab=readme) can be integrated with different web frameworks. 
 
-Also, check out [the demo site for `@airwallex/platform-onboarding-sdk`](https://static.airwallex.com/widgets/sdk-live/onboarding/demo).
+Also, check out [the demo site for `@airwallex/platform-onboarding-sdk`](https://static-demo.airwallex.com/widgets/sdk-live/onboarding/demo).
+
+## 1. Create .env in root directory
```

Comment:
for better experience, this .env file can be created for them and ask them to change the value inside